tests/ui/consts/const-float-classify.rs

```diff
@@ -1,22 +1,20 @@
 //@ compile-flags: -Zmir-opt-level=0 -Znext-solver
-//@ known-bug: #110395
-// FIXME(effects) run-pass
```

Comment:
This test got neutered in https://github.com/rust-lang/rust/pull/126552 but the test isn't about const traits at all, so I adjusted it to ensure we still have the test coverage we need, just without const traits.

Reply:
you might want to remove the const_trait_impl and effects feature from this trait if there is no plan to use that feature in this test

Reply:
I have moved re-enabling this test to https://github.com/rust-lang/rust/pull/129835